saleor/channel/migrations/0002_channel_availability.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1 on 2020-09-18 07:17
+
+from django.db import migrations, models
+
+
+def set_default_flag(apps, schema):
+    Channel = apps.get_model("channel", "Channel")
+    for channel in Channel.objects.iterator():
+        if not channel.is_active:
+            channel.is_active = True
+        channel.save(update_fields=["is_active"])
```

Comment:
Why iterate over when same can be achieved on DB level? `Channel.objects.filter(is_active=False).update(is_active=True)`?

Reply:
Furthermore - is this even needed? in line 21 we add this field and it defaults to True? Please let me know if im missing something.

Reply:
This data migration is helper for work on feature branch and it should be deleted before merge to master. I will add comment for that.